packages/common/src/store/tipping/slice.ts

```diff
@@ -104,6 +104,8 @@ const slice = createSlice({
         user: User | null
         source: TipSource
         trackId?: ID
+        onSuccessActions?: Action[]
```

Comment:
Might be worth a comment to clarify the difference between these / what they mean ^ v since it is not clear from the names

Reply:
Also 2 qs:
-why have these be actions and not just normal functions? (in which an action(s) is dispatcheD)
-why have one be an array and one be just one action?

Reply:
All very good points/concerns!
1. `onSuccessAction` is an action that fires when the tip is sent, `onSuccessConfirmedAction` is an action that fires when the tip is indexed by DN. I'll add comments!
2. It's best practice to not put functions in redux, because they're non-serializable. Although it did cause me great pain to work around that 😢 . We can definitely do it, it technically does work, but it breaks time traveling and possibly other things like rehydration etc.
3. Because I'm lazy and there's no usecases for the other one to be more than one action (yet) 😅, otherwise they'd both be arrays

Reply:
more on (2) here https://redux.js.org/faq/organizing-state#can-i-put-functions-promises-or-other-non-serializable-items-in-my-store-state